hawkeye-core/src/models.rs

```diff
@@ -17,14 +21,12 @@ pub struct Watcher {
 
 impl Watcher {
     pub fn is_valid(&self) -> Result<()> {
-        if self.slate_url.starts_with("http://")
-            || self.slate_url.starts_with("https://")
-            || self.slate_url.starts_with("file://")
-        {
-            Ok(self.source.is_valid()?)
-        } else {
-            Err(eyre!("{} not recognized as a valid URL!", self.slate_url))
-        }
+        self.transitions
+            .clone()
+            .into_iter()
```

Comment:
I don't think this needs to be cloned:

```suggestion
            .iter()
```

Reply:
great catch, I should have followed up on that one. you're absolutely right about that.